docs/source/conf.py

```diff
@@ -20,16 +20,9 @@
 # documentation root, use os.path.abspath to make it absolute, like shown here.
 #sys.path.insert(0, os.path.abspath('.'))
 
-if os.environ.get('READTHEDOCS', ''):
-    # RTD doesn't use the repo's Makefile to build docs. We run
-    # autogen_config.py to create the config docs (i.e. Configuration Options
-    # page).
-    import subprocess
-    
-    #  subprocess.run([sys.executable,'-m','pip','install','-e','../../.'])
-    
-    with open('../autogen_config.py') as f:
-        exec(compile(f.read(), 'autogen_config.py', 'exec'), {})
+# Automatically generate config_options.rst
+with open('../autogen_config.py') as f:
+    exec(compile(f.read(), 'autogen_config.py', 'exec'), {})
```

Comment:
Uhg that's an ugly line of code. This does have an issue where it uses your installed nbconvert instead of the local nbconvert. Don't feel it's required to fix for this PR though.